apps/journeys-admin/src/components/Editor/ControlPanel/Attributes/blocks/Footer/HostSidePanel/HostSidePanel.tsx

```diff
@@ -60,22 +89,39 @@ export function HostSidePanel(): ReactElement {
   }
 
   const handleSelectHost = async (hostId: string): Promise<void> => {
-    setSelectedHost(data.hosts.find((host) => host.id === hostId))
+    setSelectedHost(teamHosts.hosts.find((host) => host.id === hostId))
     setOpenSelect(false)
   }
 
   return (
     <>
       {/* DefaultHostPanel - no host */}
       <SidePanel title={t('Hosted By')} open edit>
-        {!openSelect && selectedHost == null && (
-          <SidePanelContainer border={false}>
-            <ContainedIconButton
-              label={t('Select a Host')}
-              thumbnailIcon={<UserProfiles2Icon />}
-              onClick={() => setOpenSelect(true)}
-            />
-          </SidePanelContainer>
+        {!openSelect && (selectedHost == null || !userInTeam) && (
+          <>
+            <SidePanelContainer border={!userInTeam}>
+              <ContainedIconButton
+                label={t('Select a Host')}
+                disabled={!userInTeam}
+                thumbnailIcon={<UserProfiles2Icon />}
+                onClick={() => setOpenSelect(true)}
+              />
+            </SidePanelContainer>
+            {!userInTeam && team != null && (
```

Comment:
Redundant check since this whole block only renders if `!userInTeam`?
```suggestion
            {team != null && (
```

Reply:
If I take this out, the team could exist. Eg Ziwei's team. Even if I was a member of that team, the error message would show.

I only want to show the error message when the user isn't in the team.